release/cloudbuild.yaml

```diff
@@ -78,20 +53,45 @@ steps:
       - --firmware_type=TRUSTED_APPLET
       - --tamago_version=${_TAMAGO_VERSION}
       - --raw
-      - --output_file=output/trusted_applet_manifest
-  # Print the content of the manifest.
+      - --output_file=output/trusted_applet_manifest_unsigned.json
```

Comment:
Whoops - my bad!

Reply:
No problem, it was accurate at the time :)